### DISCUSSION TITLE
Where to find subscribed hashtags

### DISCUSSION THREAD

**Discussion by daCaPo** [mastodon/mastodon#20227] [Q&A]
It's a nice feature that one can follow hashtags. Is there any place where I would see the list of all hashtags I've subscribed to?

**Comment by Terra100f** [comment]
I am also looking -- can't find it anywhere. I have typed in the same hashtag, though, and you can toggle following on and off. But otherwise, nothing.

**Comment by mdtdev** [comment]
Sadly this still appears to be an issue: there is no way to get a list of the hashtags you are following.

**Comment by zesty** [comment]
bump

**Comment by Gargron** [comment]
![image](https://user-images.githubusercontent.com/184731/225808934-711429e7-985b-48e4-bede-d96d4c2e4adf.png)
